sdk/python/kfp/compiler/_default_transformers.py

```diff
@@ -13,13 +13,21 @@
 # limitations under the License.
 
 from typing import Callable, Dict, Optional, Text
+
+from kfp.utils import telemetry
 from ..dsl._container_op import BaseOp, ContainerOp
 
 # Pod label indicating the SDK type from which the pipeline is
 # generated. By default it's set to kfp.
 _SDK_ENV_LABEL = 'pipelines.kubeflow.org/pipeline-sdk-type'
 _SDK_ENV_DEFAULT = 'kfp'
 
+# Key for component ref annotation.
+_COMPONENT_REF_ANNOTATION_KEY = 'pipelines.kubeflow.org/component_ref'
+
+# Key for component name pod label.
+_COMPONENT_NAME_LABEL_KEY = 'pipelines.kubeflow.org/component_name'
```

Comment:
I'm not sure this is really the component name...
The component name put by the author is in `component_spec.name`.

Reply:
I see. So maybe `COMPONENT_TYPE`?

Reply:
Changed to `COMPONENT_PATH`